CRM/Eventbrite/DAO/EventbriteLink.php

```diff
@@ -104,7 +104,7 @@ public static function &fields() {
         'id' => [
           'name' => 'id',
           'type' => CRM_Utils_Type::T_INT,
-          'description' => ts('Unique EventbriteLink ID'),
+          'description' => E::ts('Unique EventbriteLink ID'),
```

Comment:
Remember we don't want to bother editing DAO files; these files are managed by civix and our changes will be overwritten next time civix makes any change to the file. @lucky091588